Package.swift

```diff
@@ -24,14 +24,16 @@ let package = Package(
         ),
         .library(
             name: "SmartStore",
-            targets: ["SmartStore"]
+            targets: ["SmartStoreWrapper"]
         ),
         .library(
             name: "MobileSync",
             targets: ["MobileSync"]
         ),
     ],
     dependencies: [
+        .package(url: "https://github.com/sqlcipher/SQLCipher.swift.git", exact: "4.10.0"),
+        .package(url: "https://github.com/wmathurin/fmdb.git", branch: "spm_with_sqlcipher")
```

Comment:
NB: will switch that to forcedotcom/fmdb once https://github.com/forcedotcom/fmdb/pull/15 gets merged